http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -896,6 +896,10 @@ object RuntimeServiceInterp {
         case Some(pd) =>
           for {
             isAttached <- pd.formattedBy match {
```

Comment:
might be easier to read if we do
```
(pd.formattedBy, willByUsedBy) match {
???
}
```

Reply:
Yeah, I really don't like this block.
It seems like it can be further simplified to something like 
```
              //case Some(x) => if x is willBeUsedBy, check if its attached
              //                  else already formatted exception
              // case None => isAttached
```
I left a TODO there, but I was wary to touch this since there is a lot of disk-related work going on, and I plan on making a follow-up ticket that may completely change this logic anyways.